recipe/meta.yaml

```diff
@@ -11,20 +11,17 @@ source:
   sha256: 7135f975b28917348f3ca8feb85f83277c89ac9acf331d1b87d945ecdbe2dffb
 
 build:
-  number: 0
-  skip: true  # [py<36]
+  number: 1
+  noarch: python
```

Comment:
_every_ output also needs `noarch: python` as well. you _may_ prefer use yaml "anchors" to reduce some of the drudgery, e.g.

```yaml
build: &build
  number: 1
  noarch: python

outputs:
  - name: foo
    build: *build
```

Reply:
ah, i didn't see the `script` there.

i recommend not having any `script` in the top package, changing it to e.g. `cloudpathlib-split` and having the first output be `cloudpathlib`. You can then use the "extension" thing

```yaml
build: &build 
  number: 1 
  noarch: python

outputs:
  - name: cloudpathlib
    build:
      <<: *build
      script: {{ PYTHON }} -m pip install . -vv
```

Reply:
You'd also then need: 

```yaml
extras:
  feedstock-name: cloudpathlib
```

at the end

Reply:
I keep getting YAML parser errors that aliases are undefined when trying to use the anchors/aliases. Seems to work if I just explicitly write out the duplicate information.